### PR TITLE
docs: cleanup deps

### PR DESCRIPTION
**Pull request by peintnermax**
### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -176,6 +176,9 @@ module.exports = {
       apiKey: "124fe1c102a184bc6fc70c75dc84f96f",
       indexName: "zitadel",
       selector: "div#",
+      getMissingResultsUrl({ query }) {
+        return `https://github.com/zitadel/zitadel/discussions/new?category=ideas&title=Missing docs for ${query}`;
+      },
```
Comment (fforootd) [inline docs/docusaurus.config.js:181]:
```suggestion
```

Comment (fforootd) [reply docs/docusaurus.config.js:181]: lets leave this aside and fix the deps first (since this is not working)

**Review by fforootd — CHANGES_REQUESTED** [review]

File: docs/docusaurus.config.js (hunk 1 above)
Comment (peintnermax) [reply docs/docusaurus.config.js]: done

**Review by fforootd — APPROVED** [review]

**Comment by muhlemmer** [comment]
@peintnermax ping, merge conflicts.

**Review by fforootd — APPROVED** [review]

**Pull request merged**